<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentManagerMainWindow</class>
 <widget class="QMainWindow" name="RentManagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rent Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Companies/Properties/Units</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="trvBrowser">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="tblUnits">
       <row>
        <property name="text">
         <string>Jan</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Feb</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Mar</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Apr</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Jun</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Jul</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Aug</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Sep</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Oct</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Nov</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Dec</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Unit 1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Unit 2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Unit 3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Unit 4</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>TENANT 1</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="background">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>234</red>
           <green>45</green>
           <blue>7</blue>
          </color>
         </brush>
        </property>
       </item>
       <item row="7" column="0">
        <property name="background">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>245</red>
           <green>70</green>
           <blue>11</blue>
          </color>
         </brush>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>TENANT 3</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>TENANT 3</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionCompanies"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionUnites"/>
    <addaction name="actionTenants"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAccounting">
    <property name="title">
     <string>Accounting</string>
    </property>
    <addaction name="actionView_Tenan_Accounts"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Invoice"/>
    <addaction name="actionMulti_Invoicing"/>
    <addaction name="separator"/>
    <addaction name="actionReceive_Payments"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionAssign_Unit_To_Tenant"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManagement"/>
   <addaction name="menuAccounting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompanies"/>
   <addaction name="actionProperties"/>
   <addaction name="separator"/>
   <addaction name="actionUnites"/>
   <addaction name="actionTenants"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_Invoice"/>
   <addaction name="actionReceive_Payments"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCompanies">
   <property name="text">
    <string>Companies</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionUnites">
   <property name="text">
    <string>Units</string>
   </property>
  </action>
  <action name="actionTenants">
   <property name="text">
    <string>Tenants</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionView_Tenan_Accounts">
   <property name="text">
    <string>View Tenan Accounts</string>
   </property>
  </action>
  <action name="actionCreate_Invoice">
   <property name="text">
    <string>Create Invoice</string>
   </property>
  </action>
  <action name="actionMulti_Invoicing">
   <property name="text">
    <string>Multi Invoicing</string>
   </property>
  </action>
  <action name="actionReceive_Payments">
   <property name="text">
    <string>Receive Payments</string>
   </property>
  </action>
  <action name="actionAssign_Unit_To_Tenant">
   <property name="text">
    <string>Assign Unit To Tenant</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
